src/collection-view.js

```diff
@@ -505,17 +502,24 @@ const CollectionView = Backbone.View.extend({
 
   // Internal Method. Add the view to children and render it at the given index.
   _addChildView(view, index) {
-    monitorViewEvents(view);
-
-    // set up the child view event forwarding
-    this._proxyChildEvents(view);
+    this._setupChildView(view, index);
+    this.triggerMethod('before:add:child', this, view);
```

Comment:
alternatively if we need want to keep the public api separate from the private api we should only use the private internally.